universal-application-tool-0.0.1/app/controllers/admin/AdminProgramBlocksController.java

```diff
@@ -85,6 +88,7 @@ public Result create(Request request, long programId) {
     }
   }
 
+  /** Return a HTML page displaying all configurations of the program and forms to update them. */
```

Comment:
suggestion: maybe "displaying the configuration of the current program screen (block) and..."

Reply:
Good suggestion. Done.